README.rst

```diff
@@ -0,0 +1,260 @@
+=====================
+PyTorch Image Quality
+=====================
+
+.. image:: https://user-images.githubusercontent.com/22414094/106116361-df8a3c80-6162-11eb-83df-f5838110e614.png
```

Comment:
To unify the proposed approach, the local copy of the image from docs/_static folder should be used.